test/parallel/test-permission-deny-fs-symlink.js

```diff
@@ -0,0 +1,98 @@
+// Flags: --experimental-permission --allow-fs-read=* --allow-fs-write=*
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const fs = require('fs');
+const fixtures = require('../common/fixtures');
+const path = require('path');
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+const blockedFile = fixtures.path('permission', 'deny', 'protected-file.md');
+const blockedFolder = tmpdir.path + '/';
+const regularFile = __filename;
+const symlinkFromBlockedFile = 'example-symlink.md';
```

Comment:
I'd put this in the tmpdir to rely on the automatic cleanup. you can make `blockedFolder` a subdirectory of tmpdir.

Reply:
We can't. The system wouldn't be able to cleanup (rm) a denied folder.

Reply:
Actually, the current example shouldn't work too. This is a bug. It needs to handle trailing slashes.

Reply:
I know it wouldn't, but in case of a problem I'd prefer to have an uncleaned test/tmp.x folder instead of an uncommitted file at the top-level

Reply:
Fixed in 7173d6b7cf.